src/content/plugins/index.md

```diff
@@ -9,6 +9,7 @@ contributors:
   - eko3alpha
   - refactorized
   - byzyk
+  - fokusferit
```

Comment:
Not necessary, please add it in the pages you actually change, not just the index pages.